HTML/Task3/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Contact us</h1>
+    <p>Need an experienced and skilled hand with custom IT projects? 
+        Fill out the form to get a free consultation.</p>
+    <form>
+        <div>
+            <label for="name">Your company name<span>*</span></label>
+            <input required id="name" type="text" name="ame">
+        </div>
+        <div>
+            <label for="nature">Nature of Business<span>*</span></label>
+            <input required id="nature" type="text" name="nature">
+        </div>
+        <div>
+            <label for="date">Data of registration</label>
+            <input id="date" type="date" name="date" placeholder="DD/MM/YYYY" pattern="^(0[1-9]|1[0-2])\/([0-9]{2})$">
```

Comment:
Row 23 - Dat**e** of registration
Row 24 - placeholder and patternt not needed for input type="date"